### DISCUSSION TITLE
What resources or map layers would facilitate land-based sighting networks (CA/OR/WA/BC)?

### DISCUSSION THREAD

**Discussion by scottveirs** [salish-sea/salishsea-io#35] [Ideas]
Preliminary ideas (for 2025+ feature ideation and prioritization?):

- [ ] [List of WA, OR, CA efforts](https://whaleaware.org/whale-watching-from-land/)
- [ ] [The Whale Watching Spoken Here](http://www.oregonstateparks.org/index.cfm?do=thingstodo.dsp_whaleWatching) program which places volunteers at 26 locations in Washington, Oregon, and California
- [ ] [Google map of Oregon observation sites](https://www.google.com/maps/d/u/0/viewer?msa=0&hl=en&ie=UTF8&t=m&z=7&source=embed&mid=1GtewyAaPhNBWUDd6wbgUNClVkOM&ll=44.21568626913395%2C-124.2192685)
- [ ] [Whale Trail sites](https://thewhaletrail.org/regions/) coastwide
- [x] [Orca Network public access sites within Puget Sound](https://www.google.com/maps/d/u/0/viewer?mid=1xIsepZY5h_8oA2nd6IwJN-Y7lhk&ll=47.75952205356457%2C-122.7316404&z=8&fbclid=IwY2xjawF-iWtleHRuA2FlbQIxMAABHXrqbC74DOfTGYUhH2ohtCO0uD6vbWn0HOSHAeV0xz-ZNEUwU5XSqQGfTA_aem_grNhQpgDzXsN_EpXLc7zXw)
- [ ] Orca Behavior Institute map of public access sites and marine place names (pinned in Facebook group)

**Comment by scottveirs** [comment]
Here is an shared album of many other place names and maps used by community scientists focused on observing the marine environment -- [https://photos.app.goo.gl/rdTBfeYAvfYh9zZq9](https://photos.app.goo.gl/rdTBfeYAvfYh9zZq9)

**Comment by rainhead** [comment]
I wonder what it would take to reliably parse statements like "from Beechey Head they were 1km to the SW." The obvious answer (at this point) is LLMs, but you would need either an LLM with geospatial reasoning built in, or a prompt that produced a data structure that could be fed to code: `origin: "Beechey Head" distance: 1km bearing: SW`.

Of course, we also have to ask, can we do better with a sufficiently good UI? Will we always be extracting data from (social media) text, or will sighting networks get enough value out of salishsea.io to input sightings in a structured way?

**Comment by rainhead** [comment]
Suggestion of NOAA [ENC tiles](https://www.nauticalcharts.noaa.gov/data/gis-data-and-services.html) from Virgil. They seem useful at times, but pretty noisy more of the time.

**Comment by rainhead** [comment]
Whale Trail points are available by loading https://thewhaletrail.org/regions/ and copying `JSON.stringify(window.sites)`. There are 137.
